### PR TITLE
Fixing issue: Increase font-weight and add transitions to newsletter form submit button #10

### LINKED ISSUE #10: Increase font-weight and add transitions to newsletter form submit button
- Use a font weight of `Medium` or `Semi-Bold`.
- Add a transition for the background color

### PR DESCRIPTION
**Pull request by Pathholder1806**
Fixes #10 

Fix added in static/css/indes.css

#newsletter-form input[type="submit"] {
  font-weight: 600;
  transition: background-color 0.5s ease
}

- Taken the font-weight of 600 as **semi-bold** as 700 is the **bold**
- Added transition to the background color for ease 0.5s in accordance with other transition properties in the project

@colinkiama 
Kindly review the proposed changes and suggest any updates if you feel any!

### CODE REVIEW

File: static/css/index.css [hunk 1]
```diff
@@ -552,6 +598,11 @@ input[type="submit"]:focus {
   background-color: var(--button-hover-color);
 }
 
+#newsletter-form input[type="submit"] {
+  font-weight: 600;
```
Comment (colinkiama) [inline static/css/index.css:602]:
```suggestion
  font-weight: var(--font-weight-medium);
```

Comment (colinkiama) [reply static/css/index.css:602]: I think that the semi-bold font weight stands out to much so medium is better in this case.

**Review by colinkiama — CHANGES_REQUESTED** [review]
I'm happy to merge after the suggested changes are done.

By the way, thanks for the CSS formatting. What did you use to do it?

File: static/css/index.css (hunk 1 above)
Comment (Pathholder1806) [reply static/css/index.css:602]: Okay will surely do the same

Comment (Pathholder1806) [reply static/css/index.css]: Done @colinkiama

Comment (Pathholder1806) [reply static/css/index.css]: Done @colinkiama

**Comment by Pathholder1806** [comment]
Hey @colinkiama !
I have made the suggested changes in a new commit! Kindly review.
And I use **Prettier** as my formatter for all kinds of files.

**Comment by colinkiama** [comment]
Thanks!

**Comment by Pathholder1806** [comment]
Hey @colinkiama 
I hope your requested changes are resolved ?

**Review by colinkiama — APPROVED** [review]
I'm happy with the changes.

Sorry for the wait, I noticed a bug that I made that I didn't notice until I started reviewing this PR.

**Comment by Pathholder1806** [comment]
Yeah I saw the changes you made for resolving the bug.😀
Happy to help @colinkiama

**Pull request merged**